services/GeminiService.ts

```
@@ -0,0 +1,307 @@
+        }
+      });
+
+      const response = result.response;
+      const text = response.text();
+      
+      // Check if there's a function call
+      let functionCall = null;
+      if (response.candidates && 
+          response.candidates[0].content && 
+          response.candidates[0].content.parts && 
+          response.candidates[0].content.parts[0].functionCall) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)